examples/audio/audio_stream_effects.c

```diff
@@ -149,13 +150,17 @@ static void AudioProcessEffectLPF(void *buffer, unsigned int frames)
     static const float cutoff = 70.0f / 44100.0f; // 70 Hz lowpass filter
     const float k = cutoff / (cutoff + 0.1591549431f); // RC filter formula
 
+    // Converts the buffer data before using it
+    float * bufferData = (float *)buffer;
```

Comment:
Please, it should follow the raylib coding conventions, it should be: `float *bufferData`.

Reply:
Sorry! I generally use this convention and I forgot to change it before committing.
I'm getting used to contributing to open source, worth noting :)